pkg/network/link/ethtool.go

```diff
@@ -62,3 +64,9 @@ func EthtoolTXOff(name string) error {
 	value = EthtoolValue{ETHTOOL_STXCSUM, 0}
 	return ioctlEthtool(socket, uintptr(unsafe.Pointer(&request))) // #nosec Used for a RawSyscall
 }
+
+func closeSocket(fd int) {
```

Comment:
Could name it consistently with `closeDHCPServerIgnoringError`?

Reply:
Done.